README.md

```diff
@@ -33,24 +33,24 @@ Tightly integrated with GitLab, GitHub, and Bitbucket, Gitpod automatically and
 
 [Learn more 👉](https://www.gitpod.io/features/)
 
-Gitpod is provided as a [managed Saas version](https://gitpod.io) with a free subscription for open-source or a [free self-hosted version](https://www.gitpod.io/self-hosted/). An enterprise license is available [here](https://www.gitpod.io/self-hosted/).
+Gitpod is provided as a [managed Saas version](https://gitpod.io) with a free subscription for open-source or a [free self-hosted version](https://www.gitpod.io/self-hosted). An enterprise license is available [here](https://www.gitpod.io/self-hosted).
 
 ## Getting Started
 
 You can start using Gitpod with one or more of the following ways:
-1. [Use a Prefixed URL](https://www.gitpod.io/docs/context-urls/)
-1. [Install Browser Extension](https://www.gitpod.io/docs/browser-extension/)
-1. [Enable GitLab Integration](https://www.gitpod.io/docs/gitlab-integration/)
+1. [Use a Prefixed URL](https://www.gitpod.io/docs/getting-started/#prefixed-url)
```

Comment:
**Praise**: Nice touch! (For me, the old link still works, but the one you chose works better here 👍)

Reply:
Feels weird now, yesterday night few of the links were resulting in `404` pages on the gitpod.io side for me, specifically for the getting-started links in `README.md`. Now all of them seem to be working :no_mouth: Sorry about it.